api/v1beta1/infraenv_types.go

```diff
@@ -98,6 +98,24 @@ type InfraEnvSpec struct {
 	// +kubebuilder:default=DiscoveryImageAlways
 	// +optional
 	IPXEScriptType IPXEScriptType `json:"ipxeScriptType"`
+
+	// KernelArguments is the kernel arguments to be passed during boot time of the discovery image.
```

Comment:
These are in addition to some default kernel args?

If so, what happens if someone specifies a different value here than the default? Does this API let you override the value of a default kernel arg?

Reply:
Our implementation supports only additional arguments (for now).
The API also supports **replace** and **delete** operations.  This is meant for existing arguments manipulation.

Reply:
It would help to clarify in the comment then that these are "additional kernel arguments to be passed..."